ros/angel_system_nodes/angel_system_nodes/task_monitoring/dummy_multi_task_monitor.py

```diff
@@ -259,12 +247,16 @@ def advance_step(self, task_id: int, forward: bool) -> None:
         task_state.latest_act_classification_end_time = end_time
 
         curr_step_id = task_state.current_step_id
+        print(curr_step_id)
```

Comment:
Was this for debugging, or should this be converted into a `log.debug` message?

Reply:
Oops, good catch. Removed.